cico_setup.sh

```diff
@@ -91,10 +91,7 @@ function build_push_image() {
     fi
 
     # Build and push image
-    # Use default length when not provided
-    echo "${DEVSHIFT_TAG_LEN:=6}"
-    VERSION_NUMBER=$(echo $GIT_COMMIT | cut -c1-${DEVSHIFT_TAG_LEN})
-    TAG="SNAPSHOT-PR-${ghprbPullId}-${VERSION_NUMBER}"
+    TAG="SNAPSHOT-PR-${ghprbPullId}"
```

Comment:
"konrad assigned X,Y,Z as reviewers 14 seconds ago"

Reply:
on success build, the centos-ci user adds a comment from the job template defined. There is no way to pass values from our `cico_run_tests.sh` to `cico-jobs.yaml`. So we can not specify version number and use it in the success-comment message.

So we have removed version number and using the following command
`docker pull image_name && docker run -it -p 5000:8080 image_name` so that it pulls the latest changes made on that tag.

We can use `build number` variable provided by Jenkins ENV. but as of now, we do not see a need for maintaining each and every snapshot image created by each commit on a PR.

Reply:
@kwk It creates a new comment message for each successful build.